rocky/reports/views/generate_report.py

```diff
@@ -114,7 +114,7 @@ def post(self, request, *args, **kwargs):
             messages.error(request, _("You do not have the required permissions to enable plugins."))
```

Comment:
This check is already handled by the katalogusclient.
We should try/except the enable_boefje_by_id and catch error thrown on https://github.com/minvws/nl-kat-coordination/blob/ff6d154b9d9ba3e805c5b33e21a36125d04960db/rocky/katalogus/client.py#L366